services/firebase/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2017",
-    "skipLibCheck": true
+    "skipLibCheck": true, 
+     //"esModuleInterop":true 
```

Comment:
why is this commented?

Reply:
i was just testing it. it was not present by default